docs/getting-started/features.md

```diff
@@ -1,6 +1,8 @@
 # Features
 
-Discover all the features of the Vue DevTools
+Discover all the features of the Vue DevTools.
+
+You can open or close Vue DevTools with `Shift+Alt+D` or `Shift+Option+D` shortcut.
```

Comment:
It only works with Vite plugin, but we now have multi-client support (standalone app, browser extension)

Reply:
Maybe we can label it here: `If you are using vite plugin, you can ...` ?

Reply:
> Maybe we can label it here: `If you are using Vite plugin, you can ...` ?

@webfansplz I updated it to `If you are using Vite plugin, you can open or close Vue DevTools with `Shift+Alt+D` or `Shift+Option+D` shortcut.`, what do you think?